src/framework/json.hpp

```diff
@@ -256,6 +313,173 @@ void std::to_json(json_t &js, const std::map<uint64_t, T1, T2> &map) {
   }
 }
 
+template <typename T>
+json_t numpy_to_json_1d(py::array_t<T, py::array::c_style> arr) {
+    py::buffer_info buf = arr.request();
+    if (buf.ndim != 1) {
+        throw std::runtime_error("Number of dims must be 1");
+    }
+
+    T *ptr = (T *) buf.ptr;
+    size_t D0 = buf.shape[0];
+
+    //std::cout << "1-d conversion: " << X << std::endl;
+
+    std::vector<T> tbr;
+    for (size_t n0 = 0; n0 < D0; n0++)
+        tbr.push_back(ptr[n0]);
+
+    return std::move(tbr);
+}
+
+template <typename T>
+json_t numpy_to_json_2d(py::array_t<T, py::array::c_style> arr) {
+    py::buffer_info buf = arr.request();
+    if (buf.ndim != 2) {
+        throw std::runtime_error("Number of dims must be 2");
+    }
+
+    T *ptr = (T *) buf.ptr;
+    size_t D0 = buf.shape[0];
+    size_t D1 = buf.shape[1];
+
+    //std::cout << "2-d conversion: " << D0 << "x" << D1 << std::endl;
```

Comment:
Remove this line

Reply:
addressed in https://github.com/Qiskit/qiskit-aer/pull/399/commits/988a8e668e0ab77ce8c9b9d402d8fc889d3e0d52